app/models/designation.js

```diff
@@ -135,46 +142,56 @@ export default Model.extend({
     }
   }),
 
-  appointmentTransport: Ember.computed("orderTransport", function() {
-    let i18n = this.get("i18n");
-    return this.get("orderTransport.transportType") === "self" ?
-      i18n.t("order.appointment.self_vehicle") : i18n.t("order.appointment.hire_vehicle");
-  }),
+  appointmentTransport: Ember.computed(
```

Comment:
Unrelated to your changes, but I'm thinking we should rename properties which are translations, to something like `appointmentTransportLabel`, to make it crystal clear that it's a display string

Reply:
Sure will do.